org.libredirect.frontends_manager.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.libredirect.frontends_manager</id>
+
+  <name>Libredirect Frontends Manager</name>
+  <summary>Run privacy-friendly frontends on your computer</summary>
+
+  <url type="homepage">https://libredirect.github.io/</url>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <description>
+    <p>
+      Normally you use LibRedirect browser extension to redirect from e.g reddit.com/post =>
+      libreddit-instance.xyz/post. This does have problems like with reddit limiting requests from a
+      single ip, to you sending your data to an unkown instance. LibRedirect Frontends Manager let's
+      you to run those frontends locally on your computer. For anonymizing your IP, use Lokinet or a
+      VPN.
+    </p>
+  </description>
+  <content_rating type="oars-1.1" />
+
+  <launchable type="desktop-id">libredirect-frontends-manager.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>
+        https://raw.githubusercontent.com/libredirect/frontends_manager/main/src/assets/imgs/screenshot_1.png</image>
```

Comment:
so is the fail disappear on `main` this will break. You should use a link to the tag or commit.